openapi.yaml

```diff
@@ -0,0 +1,1883 @@
+          example: 'Line One Text'
+        lineTwo:
+          type: string
+          maxLength: 35
+          description: LineTwo
+          example: 'Line Two Text'
+        lineThree:
+          type: string
+          maxLength: 35
+          description: LineThree
+          example: 'Line Three Text'
+        lineFour:
+          type: string
+          maxLength: 35
+          description: LineFour
+          example: 'Line Four Text'
+        lineFive:
+          type: string
+          maxLength: 35
+          description: LineFive
+          example: 'Line Five Text'
+        lineSix:
+          type: string
+          maxLength: 35
+          description: LineSix
+          example: 'Line Six Text'
+        lineSeven:
+          type: string
+          maxLength: 35
+          description: LineSeven
+          example: 'Line Seven Text'
+        lineEight:
+          type: string
+          maxLength: 35
+          description: LineEight
+          example: 'Line Eight Text'
+        lineNine:
+          type: string
+          maxLength: 35
+          description: LineNine
+          example: 'Line Nine Text'
+        lineTen:
+          type: string
+          maxLength: 35
+          description: LineTen
+          example: 'Line Ten Text'
+        lineEleven:
+          type: string
+          maxLength: 35
+          description: LineEleven
+          example: 'Line Eleven Text'
+        lineTwelve:
+          type: string
+          maxLength: 35
+          description: LineTwelve
+          example: 'Line Twelve Text'
+    Address:
+      description: Address
+      properties:
+        addressLineOne:
+          type: string
+          maxLength: 35
+          description: AddressLineOne
+        addressLineTwo:
+          type: string
+          maxLength: 35
+          description: AddressLineTwo
+        addressLineThree:
+          type: string
+          maxLength: 35
+          description: AddressLineThree
+    DateMMDD:
+      properties:
+       date:
+        type: string
+        minLength: 4
+        maxLength: 4
+        description: |
+          Date
+
+          MMDD, based on the calendar date
+        example: '0401'
+    TimeHHMM:
+      properties:
+        receiptTime:
+          type: string
+          minLength: 4
+          maxLength: 4
+          description: |
+            ReceiptTime
+
+            HHMM, based on a 24-hour clock, Eastern Time
+          example: '1305'
+  # ToDo: Review ENUM's
+  # enumerations
+    TestProductionCodeEnum:
+      type: string
+      maxLength: 1
+      description: |
+        TestProductionCode
+
+        * `T` - Test
+        * `P` - Production
+      enum:
+        - T
+        - P
+      example: 'T'
+    MessageDuplicationCodeEnum:
+      type: string
+      maxLength: 1
+      description: |
+        MessageDuplicationCode
+
+        * ` ` - Original Message
+        * `R` - Retrieval of an original message
+        * `P` - Resend
+#      enum:
```

Comment:
Did you want to specify these enums?

Reply:
P is used in the TestProductionCodeEnum

OpenAPITools/openapi-generator#535